frontend/src/components/pages/teacher/DistributeAssessmentPage.tsx

```diff
@@ -17,6 +19,9 @@ const BREADCRUMB_CONFIG: BreadcrumbType[] = [
 ];
 
 const DistributeAssessmentPage = (): React.ReactElement => {
+  const { authenticatedUser } = useContext(AuthContext);
+  const { id: teacherId } = authenticatedUser ?? {};
```

Comment:
This is such a common pattern I feel like we should have created a hook for this a while back 😆 but all good now